flutter/ios/sentry_flutter.podspec

```diff
@@ -12,7 +12,7 @@ Sentry SDK for Flutter with support to native through sentry-cocoa.
                          :tag => s.version.to_s }
   s.source_files     = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
-  s.dependency 'Sentry/HybridSDK', '8.8.0'
+  s.dependency 'Sentry/HybridSDK', '8.9.1'
```

Comment:
Merge https://github.com/getsentry/sentry-dart/pull/1553/files into your branch or use it as your base branch instead.